Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM atddocker/atd-oracle-py:production
+FROM ghcr.io/oracle/oraclelinux8-instantclient:19
```

Comment:
dropping the old atd-oracle-py which doesn't work on arm64 in favor of using an oracle provided docker image. h/t @frankhereford

Reply:
good stuff. there's probably some way to alias `python=python3`.  but i've never tried this within a docker file. i'd probably just update the Airflow command out of laziness

Reply:
+1 - seems possible to add aliasing commands in this file, but it probably take some trial & error. i can review the Airflow PR to help you ship this when you need.